contrib/cdisetup/nvidia/nvidia.go

```diff
@@ -90,8 +90,11 @@ func (s *setup) Run(ctx context.Context) (err error) {
 		closeProgress(err)
 	}()
 
-	isDistro, _ := isDebianOrUbuntu()
-	if !isDistro {
+	osid, osversion, err := getOSRelease()
+	if err != nil {
+		return err
+	}
+	if osid != "debian" && osid != "ubuntu" {
```

Comment:
Looking at https://developer.download.nvidia.com/compute/cuda/repos/ we could have support for other distribs in the future.